client/containers/OrderStatus.js

```diff
@@ -8,7 +8,7 @@ const mapDispatchToProps = (dispatch) => ({
 });
 
 const mapStateToProps = (state) => ({
-  orders: state.orders,
+  orders: state.stages,
 });
 
 export default connect(mapStateToProps, mapDispatchToProps)(OrderStatus);
```

Comment:
@IgorChaikin Could you also check browser console output in development mode to fix this React warnings:

<img width="892" alt="image" src="https://user-images.githubusercontent.com/12561302/129890219-1f049eee-d1a6-470c-bc21-fa9cceeb6a6a.png">